src/Form/MessageTemplateForm.php

```diff
@@ -10,8 +10,11 @@
 use Drupal\Core\Entity\EntityForm;
 use Drupal\Core\Entity\EntityTypeInterface;
 use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
 use Drupal\message\Entity\MessageTemplate;
 use Drupal\message\FormElement\MessageTemplateMultipleTextField;
+use Drupal\message\MessagePurgePluginManager;
+use Symfony\Component\DependencyInjection\ContainerInterface;
```

Comment:
They are being used, but I found one that wasn't :)

PHPCS is running apparently, but not actually failing the build...